did.go

```diff
@@ -91,6 +100,12 @@ var DIDMethodNetwork = map[DIDMethod]map[DIDNetworkFlag]byte{
 		{Blockchain: Polygon, NetworkID: Main}:   0b00010000 | 0b00000001,
 		{Blockchain: Polygon, NetworkID: Mumbai}: 0b00010000 | 0b00000010,
 	},
+	DIDMethodShib: {
+		{Blockchain: NoChain, NetworkID: NoNetwork}: 0b00000000,
```

Comment:
DIDNetworkFlag is a structure to represent DID blockchain and network id. How are we setting network ids ?
DIDMethodShib: 

```{
		{Blockchain: NoChain, NetworkID: NoNetwork}: 0b00000000,

		{Blockchain: Shibarium, NetworkID: Main}:     0b01000000 | 0b00000001,
		{Blockchain: Shibarium, NetworkID: PuppyNet}: 0b01000000 | 0b00000010,
	}

Reply:
looks like we have same network for others as well. Is that ok?

Reply:
they're identifiers that are used in polygonID only, so we can set them up as we want as long as we don't collide with other polygonID ids... I've used the next available value in each case

Reply:
an actual reference is composed of method + blockchain + network, so we can reuse networks, there's no consequence...
iden3 uses these IDs redundantly (ie. the DID ends up specifying those twice, once in clear text, the other in binary form)